spec/features/idv/doc_auth/document_capture_spec.rb

```diff
@@ -592,6 +593,78 @@
               expect(page).to have_current_path(idv_phone_url)
             end
           end
+
+          context 'with a valid passport', driver: :headless_chrome_mobile do
+            let(:passports_enabled) { true }
+            let(:doc_auth_passport_selfie_enabled) { true }
+            let(:passport_image) do
+              Rails.root.join(
+                'spec', 'fixtures',
+                'passport_credential.yml'
+              )
+            end
+            let(:fake_dos_api_endpoint) { 'http://fake_dos_api_endpoint/' }
+
+            before do
+              allow(IdentityConfig.store).to receive(:in_person_proofing_enabled).and_return(true)
+              allow(IdentityConfig.store).to receive(:in_person_proofing_opt_in_enabled)
+                .and_return(true)
+              allow_any_instance_of(ServiceProvider).to receive(
+                :in_person_proofing_enabled,
+              ).and_return(true)
+              allow(IdentityConfig.store).to receive(:doc_auth_passports_percent).and_return(100)
+              allow(IdentityConfig.store).to receive(:dos_passport_mrz_endpoint)
+                .and_return(fake_dos_api_endpoint)
+              allow(IdentityConfig.store).to receive(:doc_auth_passport_selfie_enabled)
+                .and_return(doc_auth_passport_selfie_enabled)
+              stub_request(:post, fake_dos_api_endpoint)
+                .to_return_json({ status: 200, body: { response: 'YES' } })
+              stub_health_check_settings
+              stub_health_check_endpoints_success
+              reload_ab_tests
+            end
+
+            it 'proceeds to the next page with valid info, including a selfie image' do
+              perform_in_browser(:mobile) do
+                visit_idp_from_oidc_sp_with_ial2(facial_match_required: true)
+                sign_in_and_2fa_user(@user)
+                complete_doc_auth_steps_before_hybrid_handoff_step
+
+                choose(t('doc_auth.forms.id_type_preference.passport'))
+                click_on t('forms.buttons.continue')
+
+                expect(page).to have_current_path(idv_document_capture_url, wait: 10)
+
+                click_button 'Take photo'
```

Comment:
Can we use the i18n key instead of a string constant?
```suggestion
                click_button t('doc_auth.buttons.take_picture')
```